shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -65,6 +65,7 @@ TEST(FlutterWindowsEngine, RunDoesExpectedInitialization) {
         EXPECT_EQ(strcmp(args->dart_entrypoint_argv[1], "arg2"), 0);
         EXPECT_NE(args->platform_message_callback, nullptr);
         EXPECT_NE(args->custom_task_runners, nullptr);
+        EXPECT_NE(args->custom_task_runners->thread_priority_setter, nullptr);
```

Comment:
This needs better tests naturally...

Reply:
Can you add some tests where you call the setter function with different priority levels and check the priority of the current thread after each call?

Reply:
Yeah, when I did that I noticed I was setting process priority level 🤦

Reply:
The added test is good, but can you also call the thread_priority_setter here to make sure it's the one we think it is?

Reply:
Done